src/content/content_manager.cc

```diff
@@ -351,7 +351,9 @@ std::deque<std::shared_ptr<GenericTask>> ContentManager::getTasklist()
         return taskList;
 
     taskList.push_back(std::move(t));
-    std::copy_if(taskQueue1.begin(), taskQueue1.end(), std::back_inserter(taskList), [](auto&& task) { return task->isValid(); });
+    for (auto&& task : taskQueue1)
+        if (task->isValid())
+            taskList.push_back(std::move(task));
```

Comment:
Doesn't the have an effect on the items in `taskQueue1`?

Reply:
yeah they get removed and added to taskList.

Reply:
I think Carl is saying that as std::move doesn't actually move stuff, its just a cast, the items remain in the original collection

Reply:
hrm I don't know how true that is. The way I did this part is by temporarily making taskQueue1 and 2 unique_ptrs. It errored on std::copy_if. This loop fixed that error.